python/cugraph/cugraph/dask/structure/mg_property_graph.py

```diff
@@ -1540,8 +1544,8 @@ def renumber_edges_by_type(self, prev_id_column=None):
 
         # FIXME DASK_CUDF: https://github.com/rapidsai/cudf/issues/11795
```

Comment:
Unfortunate this bug has not been fixed yet

Reply:
Indeed. I should have a better way to work around this issue in the next release 🤞